run-clang-tidy.py

```diff
@@ -0,0 +1,503 @@
+#!/usr/bin/env python3
```

Comment:
On my machine, at least,there's already a `run-clang-tidy` shipped. I can't remember which package. What's the reason to include this here also?

Reply:
Ha, no reason. It turns out I have it too and didn't realise. I'll drop this file then